osclib/checkrepo.py

```diff
@@ -439,6 +439,12 @@ def check_specs(self, request_id=None, request=None):
         specs = [en.attrib['name'][:-5] for en in root.findall('entry')
                  if en.attrib['name'].endswith('.spec')]
 
+        # special case for glibc.i686, it have not the relevant specfile for glibc.i686
+        # but must be add it to requests list as a dummy request, otherwise the state
+        # has not be check and won't download it's binaries.
+        if 'glibc' in specs:
```

Comment:
so don't merge this...I guess it will be fail in _check_repo_download as x86_64 is excluded in glibc.i686 anyhow.